k2/csrc/util/logging.h

```diff
@@ -0,0 +1,158 @@
+// k2/csrc/util/logging_is_google_glog.h
+
+// Copyright (c)  2020  Xiaomi Corporation (authors: Meixu Song)
+
+// See ../../LICENSE for clarification regarding multiple authors
+
+/**
+ * This is the logging header that need to be included for:
```

Comment:
Hi guys, this PR intro nothing change for development as it's a glog-like one with same usage. 
Thus I put the common usages of glog here.